src/core/ext/filters/client_channel/lb_policy/weighted_target/weighted_target.cc

```diff
@@ -175,6 +175,10 @@ class WeightedTargetLb : public LoadBalancingPolicy {
       return picker_wrapper_;
     }
 
+    WeightedTargetLb* weighted_target_policy() {
```

Comment:
This method is not necessary.  `DelayedRemovalTimer` is nested inside of `WeightedChild` and can therefore access its private data members directly.

Reply:
Fixed, thanks.